src/lib/components/Card.svelte

```diff
@@ -15,6 +15,7 @@
     undefined;
   export let href: string | undefined = undefined;
   export let noPadding = false;
+  export let noMarginBottom = false;
```

Comment:
I'm wondering wouldn't the standard way to do this be to have a `--card-margin-bottom` css variable that can be overwritten by the client?

I'm not against this but I think I've seen something like that so I'm wondering if there's any consistency that we can stick to.

Reply:
Then it should be at least `--card-article-margin-bottom`. I think only having different components for different card types could simplify it.

Reply:
Used `noMargin` instead 👍